### PR TITLE
Add Fedora-based dev-containers

### PR DESCRIPTION
**Pull request by ghost**
For use within a Fedora-based environment, starting with F#.

### CODE REVIEW

File: _data/collection-index.yml [hunk 1]
```diff
@@ -613,3 +613,7 @@
   contact: https://github.com/MarlonPassos-git/dev-container-features/issues
   repository: https://github.com/MarlonPassos-git/dev-container-features
   ociReference: ghcr.io/marlonpassos-git/dev-container-features
+- name: DevContainers for Fedora-based Development
+  maintainer: RonaldTM9999
+  contact: https://github.com/RonaldTM9999/devcontainers/issues
```
Comment (samruddhikhandale) [inline _data/collection-index.yml:618]: Can you help add the `repository` property? Thanks!

File: _data/collection-index.yml [hunk 2]
```diff
@@ -613,3 +613,7 @@
...
+  ociReference: https://ghcr.io/RonaldTM9999/devcontainers
```
Comment (samruddhikhandale) [inline _data/collection-index.yml]:
```suggestion
  ociReference: ghcr.io/ronaldtm9999/devcontainers
```

Hi 👋

The ORAS spec requires the ociReference to be in lower case. Can we update here along with the removal of `https` prefix, thanks?
The supporting tools do help convert the case, however, its better to sync as much as possible.

File: _data/collection-index.yml (hunk 1 above)
Comment (ghost) [reply _data/collection-index.yml:618]: Repository added, thank you for catching that!

**Comment by ghost** [comment]
Thank you for the review, @samruddhikhandale. I've addressed the issues you've raised.

**Review by samruddhikhandale — DISMISSED** [review]
Looks great, thank you!

**Review by samruddhikhandale — APPROVED** [review]

**Pull request merged**